todo_app/app.py

```diff
@@ -1,11 +1,20 @@
-from flask import Flask
+from flask import Flask, redirect, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items=items)
+
+@app.route('/item', methods=["POST"])
+def create_item():
+    item_title = request.form.get('item_title')
+    print(item_title)
```

Comment:
This is useful for debugging, but is probably worth clearing up now that you've got it working rather than littering the logs with just this